README.md

```diff
@@ -10,38 +10,79 @@ neural network (GNN) that predicts an association to a target molecule, e.g., a
 
 You have several ways to provide the correct environment to run code from the DFPL package.
 
-1. Use the automatically built docker/Singularity containers
-2. Build your own container [following the steps here](container/README.md)
-3. Setup a python virtual environment
-4. Set up a conda environment install the requirements via conda and the DFPL package via pip
+1. Use bioconda to install the package
+2. Set up a Python virtual environment
+3. Use the automatically built Docker
+4. Use the automatically built Singularity containers
 
-In the following, you find details for option 1., 3., and 4.
+### Bioconda
+
+The package is also available on Bioconda. You can find the Bioconda recipe here and
+[![install with bioconda](http://bioconda.github.io/recipes/deepfplearn/README.html)]
+
+First create an environment with Python 3.8:
+
+```shell
+conda create -n dfpl python=3.8
+conda activate dfpl
+```
+
+Then install the package:
+
+```shell
+conda install -c bioconda deepfplearn
+```
+
+### Set up DFPL in a python virtual environment
+
+From within the `deepFPlearn` directory call
+
+```
+virtualenv -p python3 ENV_PATH
+. ENV_PATH/bin/activate
+pip install ./
+```
+
+replace `ENV_PATH` by the directory where the python virtual environment should be created.
+If your system has only python3 installed `-p python3` may be removed.
+
+In order to use the environment, it needs to be activated with `. ENV_PATH/bin/activate`.
 
 ### Docker container
 
-You need docker installed on you machine.
+You need docker installed on your machine. If you don't have it installed yet, you can find the installation
+instructions [here](https://docs.docker.com/engine/install/).
+
+In order to run DFPL pull the image using the following command line:
 
-In order to run DFPL use the following command line
+```shell
+docker pull quay.io/biocontainers/deepfplearn:TAG
+```
+Then mount the directory containing the data you want to process and run the container with the following command:
 
 ```shell
-docker run --gpus GPU_REQUEST registry.hzdr.de/department-computational-biology/deepfplearn/deepfplearn:TAG dfpl DFPL_ARGS
+docker run -v /path/to/local/repo quay.io/biocontainers/deepfplearn:TAG dfpl DFPL_ARGS
+```
+And then you can run the container with the following command:
+
+```shell
+docker run --gpus GPU_REQUEST quay.io/biocontainers/deepfplearn:TAG dfpl DFPL_ARGS
 ```
 
 where you replace
 
-- `TAG` by the version you want to use or `latest` if you want to use latest available version)
-- You can see available tags
-  here https://gitlab.hzdr.de/department-computational-biology/deepfplearn/container_registry/5827.
+- `TAG` by the version you want to use
+- You can see available tags in [biocontainers](https://biocontainers.pro/tools/deepfplearn).
   In general a container should be available for each released version of DFPL.
 - `GPU_REQUEST` by the GPUs you want to use or `all` if all GPUs should be used (remove `--gpus GPU_REQUEST` if only the
```

Comment:
GPUs probably do not work yet in the container.

Reply:
I will remove them for now.